test/headful.spec.js

```diff
@@ -76,6 +76,14 @@ module.exports.addTests = function({testRunner, expect, defaultBrowserOptions})
       expect(await page.evaluate(() => 2 * 3)).toBe(6);
       await browserWithExtension.close();
     });
+    it('target.title() should return extension title', async({}) => {
+      const browserWithExtension = await puppeteer.launch(extensionOptions);
+      const backgroundPageTarget = await waitForBackgroundPageTarget(browserWithExtension);
+      const backgroundPageTitle = backgroundPageTarget.title();
+      expect(backgroundPageTitle === 'Simple extension' ||
+        backgroundPageTitle === 'CryptoTokenExtension').toBeTruthy();
```

Comment:
At least on my machine, these tests actually load two extensions, `CryptoTokenExtension`, and `Simple extension`, and  `CryptoTokenExtension` looks like it's the one actually always passed back by `waitForBackgroundPageTarget` (which makes sense since it's presumably built in and loads first). Depending on when the extensions load relative to the test, after the `await`, `browser.targets()` sometimes has `Simple extension` and sometimes doesn't.

None of these tests seem to be dependent on exactly which extensions' `background_page` is being returned, so that seems fine, but it seemed like a bad idea to depend on `CryptoTokenExtension` existing forever (and appearing in this order) for this test. I could also modify `waitForBackgroundPageTarget` to be more "wait for a *particular* background target" instead of checking both titles if that's better.